templates/components/what/wasm/production.hbs

```diff
@@ -1,62 +1,60 @@
 <section id="production" class="white">
     <div class="w-100 mw-none ph3 mw8-m mw9-l center f3">
         <header>
-            <h2>Production use</h2>
+            <h2>{{text wasm-production-heading}}</h2>
             <div class="highlight"></div>
         </header>
         <div class="testimonials">
             <div class="testimonial flex flex-row">
                 <div class="w-30-l tc pa5">
-                  <img src="/static/images/user-logos/cloudflare.svg" alt="cloudflare logo" />
+                  <img src="/static/images/user-logos/cloudflare.svg" alt="{{text wasm-production-cloudflare-alt}}" />
                 </div>
                 <div class="w-70-l">
                     <blockquote>
-                       We can compile Rust to WASM, and call it from Serverless functions woven into the very fabric of the Internet. That’s huge and I can’t wait to do more of it.
+                       {{text wasm-production-cloudflare-quote}}
                     </blockquote>
                     <p class="attribution">
-                        &ndash;  Steven Pack,
-                        <a href="https://blog.cloudflare.com/cloudflare-workers-as-a-serverless-rust-platform/">
-                            Serverless Rust with Cloudflare Workers
-                        </a>
-
+                        {{#text wasm-production-cloudflare-attribution}}
+                            {{#textparam href}}
```

Comment:
This can just be `{{text wasm-production-cloudflare-attribution href="https://..."}}`, textparam is for more complex stuff